tests/cpp/tdont_init_instantiation.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   targets: "cpp"
   output: ''''''
-  disabled: true
+    disabled: true
```

Comment:
References  https://github.com/nim-lang/Nim/issues/5140
		Disabled around 2016-12, for CI sake, it seems.

Reply:
If this does not work properly, we can just put a "known issue" on the most Cxx interop - I mean, if nim clobbers the init data of the class constructor, it is really bad.

Reply:
Added a knownIssue, pointing to the issue it refered to when created (#5140)